next/app/components/layouts/Header/Header.module.css

```
@@ -0,0 +1,23 @@
+.header {
+  display: flex;
+  background-color: #edc3e7;
+  font-size: 36px;
```

Figamのデザイン通りでグッドです！
実際に表示すると、少し大きめなので小さくしてほしいです！
```suggestion
  font-size: 30px;
```